### PR TITLE
Update the symbol names for all ConfigErrors to adopt camelCase consi…

### PR DESCRIPTION
**Pull request by geeknoid**
…stently.

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/625)

### CODE REVIEW

**Comment by ZackButcher** [comment]
Wait, I thought we were using proto field names (so snake_case)?

**Comment by geeknoid** [comment]
I'm pretty sure we settle on camelCase for our docs, which is what I
changed everything to last week. So these field names should be consistent
with that I think. Did we agree on snake_case and I'm misremembering?

**Comment by ZackButcher** [comment]
The only thing I recall is that we wanted field names to match exactly to make debugging easier. However, I'm also planning on transforming our auto-gen'd docs to use camelCase. The only place snake_case names will exist is in the .proto files. Given that, this is fine I think.

File: out.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+//adapter/denyChecker:small_tests                               (cached) PASSED in 0.1s
```
Comment (douglas-reid) [inline out.txt]: please remove from PR.

**Comment by douglas-reid** [comment]
I had thought we decided to go with snake_case, too (which explains all those lines in kubernetes.go you had to change). I really don't care much either way. However, I would like to have this definitively decided for moving forward.

Can we add some doc somewhere (maybe in ConfigErrors) that indicates (or even enforces) the expected style?

**Comment by kyessenov** [comment]
camelCase is standard in JSON/YAML and that's what the operator sees. That's how k8s YAMLs look like. Unless there is a reason to go with the snake case, let's utilize camelCase consistently.

**Pull request merged**